### DISCUSSION TITLE
Maximize Icon in mobile is missing

### DISCUSSION THREAD

**Discussion by heylucifurr** [DonutWare/Fladder#106] [General]
There is no maximize button on mobile screen using ios.

**Comment by PartyDonut** [comment]
The full-screen or fill video for mobile is either by using gestures (pinch-zoom) or by going to  Settings->Player and then enabling Fill screen. This should make the video expand below the notches as well.

Because of space constraints the maximize button is not added for mobile. And because it strange for mobile to have a maximize button the same way the other platforms have.

**Comment by heylucifurr** [comment]
tried it. but im not sure why on my phone its not working well. currently heres what it appears on landscape mode.
https://imgur.com/a/61CN8MU
On android (samsung s21 ultra 'sasmung internet browser') it works alright. I can see pop up buttons of Maximize and PIP

**Comment by PartyDonut** [comment]
Oh you're using the Fladder browser version on a phone. I was not expecting that combination 😄 .

It should show the button however it looks like it's cut-off because of the smaller screen size. I can have a look at it but that's probably very low priority.

Out of curiosity why not use the native application for mobile?

**Comment by heylucifurr** [comment]
Yeah I figured thats the case, coz I saw the icon cutoff on the ipad.

I did have the native app of jellyfin. But honetly I dont like the UI of it.  so I plugged my Flabber into a domain and whats where my family watch on any device.

**Reply by PartyDonut** [reply]
I mean as in the native version of Fladder.

Fladder is build for most platforms including iOS and Android.

**Comment by heylucifurr** [comment]
Yeah I figured thats the case, coz I saw the icon cutoff on the ipad.

I did have the native app of jellyfin and white labelled it. BUt honetly I dont like the UI of it.  so I plugged my Flabber into a domain and whats where my family watch on any device.
